blog/_src/posts/2019-11-15-completing-racket-s-relicensing-effort.md

```diff
@@ -0,0 +1,80 @@
+value both that community and the ability to build on other free
+software. However, two factors mean have made our previous license,
+the GNU LGPL, less of a good fit going forward. First, it is unclear
+how to apply the LGPL's statement about dynamic linking to a language
+like Racket, where macro expansion can copy code from libraries to
+applications, and where applications are typically bundled with the
+Racket runtime and libraries. Second, some organizations unfortunately
+are unwilling to use software licensed under any variant of the
+GPL. Since we want to provide a clear license and to promote the use
+of Racket everywhere, a new, more-permissive license is the right
+choice for Racket.
+
+## What is available under the new license?
+
+In the soon-to-be-released Racket 7.5, the Racket CS binary, and any
+Racket code in the Minimal Racket distribution, is available under
+both the Apache 2.0 and MIT licenses. Additionally, almost all of the
+packages distributed with the full Racket distribution are available
+under these licenses. The exceptions are
+["slatex"](https://github.com/racket/slatex/blob/master/LICENSE),
+["srfi-doc"](https://github.com/racket/srfi/blob/master/LICENSE),
+["swindle"](https://github.com/racket/swindle/blob/master/LICENSE),
+["r5rs-doc"](https://github.com/racket/r5rs/blob/master/LICENSE),
+["r6rs-doc"](https://github.com/racket/r6rs/blob/master/LICENSE), and
+["r6rs-std-doc"](https://github.com/racket/r6rs/blob/master/LICENSE)
+as well as "string-constants-lib" which is discussed below; see the
+LICENSE files available in those individual packages for details.  The
+traditional Racket binary itself, as well as other executables such as
+the DrRacket binary, continues to include code from several libraries
+distributed under the LGPL and is thus still distributed with that
+license; we do not anticipate this changing.
+
+Additionally, packages that include the compiled version of [native
+libraries](https://github.com/racket/libs/#license) such as Gtk which
+themselves are licensed under the LGPL retain that license.
+
+Some packages did not complete the re-licensing process until after
+the release process for Racket 7.5 began; those packages will ship
+with new licenses in Racket 7.6.
+
+## Is there anything remaining to re-license?
+
+We've completed the re-licensing for all code that we planned to
+cover. However, some translations (part of the ["string-constants-lib"
+package](https://github.com/racket/string-constants/blob/master/LICENSE))
+have authors that are unknown or have not been responded to us. This
+does not affect the licensing of any programs using Racket unless they
+depend on or derive from those translations directly.
+
+## Did anyone say no?
+
+Two people declined to re-license their contributions to Racket. We
+therefore removed their contributions and, where appropriate, replaced
+them with new code and/or documentation.
+
+<!-- more -->
```

Comment:
Is this intentional? (Makes the entire post appear on pages that list posts + abstracts)